src/main/scala/org/apache/sysml/api/dl/CaffeLayer.scala

```diff
@@ -1032,9 +1134,14 @@ class DeConvolution(val param:LayerParameter, val id:Int, val net:CaffeNetwork)
    *  - Hout: Output height.
```

Comment:
Can you update `out` here too with/without depthwise?  The depthwise version is:

```
*  - out: Outputs, of shape (N, C/M*Hout*Wout).
```